src/elements/file-selector/file-selector.stories.ts

```diff
@@ -175,12 +229,39 @@ export const DefaultOnlyPDF: StoryObj = {
   args: { ...defaultArgs, accept: '.pdf' },
 };
 
+export const DefaultMultiSizeS: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...multipleDefaultArgsSizeS },
+  play: isChromatic() ? ({ canvasElement }) => playStory(canvasElement, 3) : undefined,
+};
+
+export const DefaultDisabledSizeS: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgsSizeS, disabled: true },
+};
+
+export const DropzoneMultiSizeS: StoryObj = {
```

Comment:
As discussed, potentially reduce amounts of tests

Reply:
Removed the disabled size s and kept just two stories for the two variants (default/dropzone).